app/controllers/report_controller/usage.rb

```diff
@@ -56,27 +56,22 @@ def usage_option_chooser
     session[:usage_options] = @usage_options
 
     render :update do |page|
-#     if new_toolbars
-        if @usage_options[:report] && @usage_options[:report].table.data.length > 0
-          page << "ManageIQ.toolbars.center_tb.obj.showItem('usage_txt');"
-          page << "ManageIQ.toolbars.center_tb.obj.enableItem('usage_txt');"
-          page << "ManageIQ.toolbars.center_tb.obj.showItem('usage_csv');"
-          page << "ManageIQ.toolbars.center_tb.obj.enableItem('usage_csv');"
-          page << "ManageIQ.toolbars.center_tb.obj.showItem('usage_pdf');"
-          page << "ManageIQ.toolbars.center_tb.obj.enableItem('usage_pdf');"
-          page << "ManageIQ.toolbars.center_tb.obj.showItem('usage_reportonly');"
-          page << "ManageIQ.toolbars.center_tb.obj.enableItem('usage_reportonly');"
-        else
-          page << "ManageIQ.toolbars.center_tb.obj.hideItem('usage_txt');"
-          page << "ManageIQ.toolbars.center_tb.obj.hideItem('usage_csv');"
-          page << "ManageIQ.toolbars.center_tb.obj.hideItem('usage_pdf');"
-          page << "ManageIQ.toolbars.center_tb.obj.hideItem('usage_reportonly');"
-        end
-#     else
-#       page.replace_html("center_buttons_div", :partial=>"layouts/center_buttons")
-#     end
+      if @usage_options[:report] && @usage_options[:report].table.data.length > 0
```

Comment:
@h-kataria `@usage_options` is being used here, while you want to remove it in a previous comment.  Should this whole `if` statement simply go away here?